app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftServiceImpl.java

```diff
@@ -75,10 +74,21 @@
 
     private final NamespacedOpenShiftClient openShiftClient;
     private final OpenShiftConfigurationProperties config;
+    private Map<String, String> productizedLabelsTemplate = new HashMap<>();
 
     public OpenShiftServiceImpl(NamespacedOpenShiftClient openShiftClient, OpenShiftConfigurationProperties config) {
         this.openShiftClient = openShiftClient;
         this.config = config;
+        // retrieve the productized labels: com.redhat, they exists only when syndesis-operator is built with productized=true
+        DeploymentConfig dc = openShiftClient.deploymentConfigs() != null  ?
```

Comment:
Just a note to say this will be modified when the kubernetes branch gets merged since all deploymentconfigs will become deployments.
Will rebase the k8 branch to include this then modify. Will be couple of days yet before merging.

Reply:
ok, once you branch is merged I will change this impl.